app/resources/view.html.template

```diff
@@ -54,32 +54,31 @@
 
       <div class="column start">
         <div class="fields-table person">
+          <div class="header section">
+            <h1>{{person.full_name_list|first}}
+          </div>
           <div>
 
             <h2>
-              {% trans "Identifying information" %}
               {% if person.provider_name %}<br />
               ({% trans "Provided by:" %} {{person.provider_name}})
               {% endif %}
             </h2>
 
             <div class="name section">
-              <h3>{% trans "Name" %}</h3>
-              <div class="field">
-                <span class="label">{% trans "Full name" %}:</span>
-                <span class="value">{{person.full_name_list|join:"<br>"}}</span>
-              </div>
+              <h2>{% trans "Identifying Information" %}</h2>
+              {% if person.full_name_list|length > 2 %}
```

Comment:
This whole full name section doesn't need to be here since the full name is now at the top. I must have made a mistake while doing this and then not caught it since it will never be greater than 2.
Removed.